### PR TITLE
Reworked README.md - readability, spelling, flow, links

### PR DESCRIPTION
**Pull request by aaronxn**
Removed NixNote info (until it's fixed) See: https://discourse.joplinapp.org/t/evernote-nixnote2-joplin/6665

Removed Firefox Web Clipper info (until it's fixed / available)

### CODE REVIEW

**Comment by seanieb** [comment]
I don't think we should "correct" british spelling.  The rest of the edits are welcome, but he GB->US isn't helpful.

File: README.md [hunk 1]
```diff
@@ -6,33 +6,35 @@
...
 -----------------|----------|----------------
-Android          | <a href='https://play.google.com/store/apps/details?id=net.cozic.joplin&utm_source=GitHub&utm_campaign=README&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' height="40px" src='https://joplinapp.org/images/BadgeAndroid.png'/></a> | or download the APK file: [64-bit](https://github.com/laurent22/joplin-android/releases/download/android-v1.0.320/joplin-v1.0.320.apk) [32-bit](https://github.com/laurent22/joplin-android/releases/download/android-v1.0.320/joplin-v1.0.320-32bit.apk)
+Android          | <a href='https://play.google.com/store/apps/details?id=net.cozic.joplin&utm_source=GitHub&utm_campaign=README&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' height="40px" src='https://joplinapp.org/images/BadgeAndroid.png'/></a> | or download the APK file: [64-bit](https://github.com/laurent22/joplin-android/releases/download/android-v1.0.319-rc1/joplin-v1.0.319-rc1.apk) [32-bit](https://github.com/laurent22/joplin-android/releases/download/android-v1.0.319-rc1/joplin-v1.0.319-rc1-32bit.apk)
```
Comment (tessus) [inline README.md]: Some of the README.md is automatically generated and replaced, so I do not know, if changes to that part will persist.

**Comment by laurent22** [comment]
As seanieb said, please revert the changes to British spelling.

File: README.md (hunk 1 above)
Comment (laurent22) [reply README.md]: Indeed the links are auto-generated.

**Comment by aaronxn** [comment]
> As seanieb said, please revert the changes to British spelling.

Will do.

FYI: I was using my spell check and just fixed the 'misspelled' words it pointed out. I'm in the usa, so it uses American English spelling. Soooooo, you can see what happened. Ugh.

**Pull request closed without merging**